model/task/task.go

```diff
@@ -4060,7 +4089,7 @@ func HasMatchingTasks(versionID string, opts HasMatchingTasksOptions) (bool, err
 }
 
 func getTasksByVersionPipeline(versionID string, opts GetTasksByVersionOptions) ([]bson.M, error) {
-	var match bson.M = bson.M{}
+	var match = bson.M{}
```

Comment:
nit: `match := bson.M{}`

Reply:
blame my editor for this change :fistshake: